website/index.html

```diff
@@ -510,6 +518,30 @@ <h2>Like ClickHouse?</h2>
         if (hostParts.length > 2 && hostParts[0] != 'test') {
             window.location.host = hostParts[0] + '.' + hostParts[1];
         }
+
+        var available_distributives = ['deb', 'rpm'];
```

Comment:
it's more like package formats rather than distributives
also maybe let's also add tar.gz?

Reply:
.tar.gz are not ready.